src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1 +1,40 @@
-export const TodoInfo = () => {};
+import classNames from 'classnames';
```

Comment:
Consider using named imports instead of default imports for better readability and maintainability. 🤓